sormas-ui/src/main/java/de/symeda/sormas/ui/task/BulkTaskDataForm.java

```diff
@@ -0,0 +1,161 @@
+		+ fluidRowLocsCss(VSPACE_4, STATUS_CHECKBOX)
+		+ fluidRowLocs(TaskBulkEditData.TASK_STATUS);
+
+	private boolean initialized = false;
+
+	private CheckBox reassignedToCheckbox;
+	private CheckBox priorityCheckbox;
+	private CheckBox taskStatusCheckbox;
+
+	public BulkTaskDataForm() {
+		super(TaskBulkEditData.class, TaskDto.I18N_PREFIX);
+		setWidth(680, Unit.PIXELS);
+		hideValidationUntilNextCommit();
+		initialized = true;
+		addFields();
+	}
+
+	@Override
+	protected void addFields() {
+
+		if (!initialized) {
+			return;
+		}
+
+		taskStatusCheckbox = new CheckBox(I18nProperties.getCaption(Captions.bulkTaskStatus));
+		getContent().addComponent(taskStatusCheckbox, STATUS_CHECKBOX);
+		NullableOptionGroup status = addField(TaskBulkEditData.TASK_STATUS, NullableOptionGroup.class);
+		status.setEnabled(false);
+		FieldHelper.setRequiredWhen(getFieldGroup(), taskStatusCheckbox, Arrays.asList(TaskBulkEditData.TASK_STATUS), Arrays.asList(true));
+		taskStatusCheckbox.addValueChangeListener(e -> {
+			status.setEnabled((boolean) e.getProperty().getValue());
+		});
+
+		priorityCheckbox = new CheckBox(I18nProperties.getCaption(Captions.bulkTaskPriority));
+		getContent().addComponent(priorityCheckbox, PRORITY_CHECKBOX);
+		NullableOptionGroup priority = addField(TaskBulkEditData.TASK_PRIORITY, NullableOptionGroup.class);
+		priority.setEnabled(false);
+		FieldHelper.setRequiredWhen(getFieldGroup(), priorityCheckbox, Arrays.asList(TaskBulkEditData.TASK_PRIORITY), Arrays.asList(true));
+		priorityCheckbox.addValueChangeListener(e -> {
+			priority.setEnabled((boolean) e.getProperty().getValue());
+		});
+
+		reassignedToCheckbox = new CheckBox(I18nProperties.getCaption(Captions.bulkTaskReassignedTo));
+		getContent().addComponent(reassignedToCheckbox, REASSIGNED_TO_CHECKBOX);
+		ComboBox reassignedTo = addField(TaskBulkEditData.TASK_REASSIGNED_TO, ComboBox.class);
+		reassignedTo.setEnabled(false);
+		FieldHelper.addSoftRequiredStyleWhen(
+			getFieldGroup(),
+			reassignedToCheckbox,
+			Arrays.asList(TaskBulkEditData.TASK_REASSIGNED_TO),
+			Arrays.asList(true),
+			null);
+
+		List<UserReferenceDto> users = getUsers();
+		Map<String, Long> userTaskCounts =
+			FacadeProvider.getTaskFacade().getPendingTaskCountPerUser(users.stream().map(ReferenceDto::getUuid).collect(Collectors.toList()));
+		for (UserReferenceDto user : users) {
+			reassignedTo.addItem(user);
+			Long userTaskCount = userTaskCounts.get(user.getUuid());
+			reassignedTo.setItemCaption(user, user.getCaption() + " (" + (userTaskCount != null ? userTaskCount.toString() : "0") + ")");
+		}
+
+		reassignedToCheckbox.addValueChangeListener(e -> {
+			reassignedTo.setEnabled((boolean) e.getProperty().getValue());
+		});
+	}
+
+	private List<UserReferenceDto> getUsers() {
+		final List<UserReferenceDto> users = new ArrayList<>();
+		UserDto userDto = UserProvider.getCurrent().getUser();
+
+		users.addAll(FacadeProvider.getUserFacade().getAllUserRefs(false));
```

Comment:
This will allow users to assign tasks to users that normally wouldn't be allowed to be assigned, which is not okay. This logic either needs to replicate the logic from TaskEditForm or it can't be possible to bulk-assign tasks to new users. If that's a necessity, I guess assigning new users to tasks can only be possible if e.g. only tasks associated with cases from the same district are selected, etc.